### ISSUE TITLE
Question about model uncertainty of non-bayesian model when training

### ISSUE THREAD

**Issue by lisosia** [cai-mj/UMA#3]
Hi, Thank you for your great work.

When training for non-bayesian model, 
it seems that uncertainty map is calculated by `softmax(logit)`:
https://github.com/cai-mj/UMA/blob/b72dad96ce9e219adbf70bc2f119af65f01bec84/train_bayesian_adapt.py#L359-L362

it it more reasonable to use something like `abs(softmax(logit) - 0.5)` ?
It seems that Figure2(c) shows something like the latter equation to calc uncertainty map.

Thanks.

**Comment by cai-mj** [comment]
> Hi, Thank you for your great work.
> 
> When training for non-bayesian model,
> it seems that certainty map is calculated by `softmax(logit)`:
> https://github.com/cai-mj/UMA/blob/b72dad96ce9e219adbf70bc2f119af65f01bec84/train_bayesian_adapt.py#L359-L362
> 
> Is it more reasonable to use `abs(softmax(logit) - 0.5)` ?
> It seems that Figure2(c) use something like the latter equation to calc (un)certainty map.
> 
> Thanks.

Hi, my code doesn't calculate uncertainty map for non-bayesian model. The comment before that four lines of code is mistakenly written (sorry).
The Figure 2(c) in the paper was indeed visualized based on `uncertainty = 1 - abs(softmax(logit) - 0.5)*2`  for demonstration.

**Comment by lisosia** [comment]
Thanks for your clarification.
Is the equation `uncertainty = 1 - abs(softmax(logit) - 0.5)*2` also used for the experimant of Table1 (CNN+uma) ?
